supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -52,22 +61,41 @@ class JesInitializationActor(override val workflowDescriptor: BackendWorkflowDes
   /**
     * Abort all initializations.
     */
+  // TODO PBE ??? is not cool
   override def abortInitialization(): Unit = ???
```

Comment:
If it does I find it scary that we're just silently eating exceptions - it means we're back in the world where we're managing to both not let it crash but also not doing alerting anything to misbehavior.